docker/run_app

```diff
@@ -3,6 +3,6 @@ set -ex
 HERE=$(dirname $0)
 . $HERE/common
 
-export MINT_BRANCH=mrc-4049
+export MINT_BRANCH=master
 
-docker-compose up -d
\ No newline at end of file
+docker-compose up -d --quiet-pull
```

Comment:
I think the problem I was seeing with disappearing graph may have been becaise this wasn't pulling the latest images - I thought docker-compose did that by default. I think this should work?

Reply:
that makes sense. I think the default is to pull only if a tag is not present, which makes sense with the issues you report